### PR TITLE
Change to new SDK-style project; updated analyzers, changed to net472.

### PR DESCRIPTION
**Pull request by alistairjevans**

### CODE REVIEW

File: src/Autofac.Integration.WebApi/Autofac.Integration.WebApi.csproj [hunk 1]
```diff
@@ -1,138 +1,65 @@
...
+    <RepositoryUrl>https://github.com/autofac/Autofac.WebApi</RepositoryUrl>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <PackageIcon>icon.png</PackageIcon>
+    <PackageProjectUrl>https://autofac.org</PackageProjectUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageReleaseNotes>Release notes are at https://github.com/autofac/Autofac.WebApi/releases</PackageReleaseNotes>
+    <Features>IOperation</Features>
+    <PackageId>Autofac.WebApi2</PackageId>
+    <AssemblyName>Autofac.Integration.WebApi</AssemblyName>
+    <Company>Autofac Project - https://autofac.org</Company>
```
Comment (tillig) [inline src/Autofac.Integration.WebApi/Autofac.Integration.WebApi.csproj:18]:
```suggestion
    <Company>Autofac</Company>
    <Authors>Autofac Contributors</Authors>
```
Consistent with core Autofac.

File: src/Autofac.Integration.WebApi/Autofac.Integration.WebApi.csproj [hunk 2]
```diff
@@ -1,138 +1,65 @@
...
   <ItemGroup>
-    <PackageReference Include="Autofac">
-      <Version>5.2.0</Version>
-    </PackageReference>
-    <PackageReference Include="Microsoft.AspNet.WebApi.Core">
-      <Version>5.2.0</Version>
-    </PackageReference>
-    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers">
-      <Version>2.6.1</Version>
+    <PackageReference Include="Autofac" Version="[5.0.0, 7.0.0)" />
```
Comment (tillig) [inline src/Autofac.Integration.WebApi/Autofac.Integration.WebApi.csproj:48]: [In Autofac.Mvc we ended up going with straight 6.0+.](https://github.com/autofac/Autofac.Mvc/blob/70c77d1485b00797332560fa31e7a435ed4a63f6/src/Autofac.Integration.Mvc/Autofac.Integration.Mvc.csproj#L34) Should we do that here, too?

File: global.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "3.1.301"
```
Comment (tillig) [inline global.json:3]:
```suggestion
    "version": "3.1.301",
    "rollForward": "latestFeature"
```

**Review by tillig — CHANGES_REQUESTED** [review]
Couple minor things, otherwise good to go.

File: src/Autofac.Integration.WebApi/Autofac.Integration.WebApi.csproj (hunk 2 above)
Comment (alistairjevans) [reply src/Autofac.Integration.WebApi/Autofac.Integration.WebApi.csproj:48]: Yes, we should be consistent, especially among the tightly couple Mvc+WebApi projects. Will fix.

**Review by tillig — APPROVED** [review]
🌜

**Pull request merged**